docker-compose.yml

```diff
@@ -1,18 +1,43 @@
-version: "3"
+version: "3.8"
 
 services:
   web:
-    image: dhis2/core-dev:local
+    image: "${DHIS2_IMAGE:-dhis2/core-dev:local}"
     ports:
       - "8080:8080"
     volumes:
       - ./docker/dhis.conf:/opt/dhis2/dhis.conf:ro
     depends_on:
-      - db
+      db:
+        condition: service_healthy
 
   db:
     image: postgis/postgis:10-2.5-alpine
+    volumes:
+      - db-dump:/docker-entrypoint-initdb.d/
     environment:
       POSTGRES_USER: dhis
       POSTGRES_DB: dhis2
-      POSTGRES_PASSWORD: dhis
+      POSTGRES_PASSWORD: &postgres_password dhis
+      PGPASSWORD: *postgres_password # needed by psql in healthcheck
+    healthcheck:
+      test: ["CMD-SHELL", "psql --no-password --quiet --username $$POSTGRES_USER postgres://127.0.0.1/$$POSTGRES_DB -p 5432 --command \"SELECT 'ok'\" > /dev/null"]
+      start_period: 120s
+      interval: 1s
+      timeout: 3s
+      retries: 5
+    depends_on:
+      db-dump:
+        condition: service_completed_successfully # make sure the DB dump has been downloaded
+
+  db-dump:
+    image: busybox # busybox wget version does not have --no-clobber, so we need to do the [ -f ] test
+    command: sh -c '[ -f dump.sql.gz ] && echo "dump.sql.gz exists" || wget --output-document dump.sql.gz $$DHIS2_DB_DUMP_URL' # only download file if it does not exist
+    environment:
+      DHIS2_DB_DUMP_URL: "${DHIS2_DB_DUMP_URL:-https://databases.dhis2.org/sierra-leone/2.38.1/dhis2-db-sierra-leone.sql.gz}"
```

Comment:
Should the default version here be `2.38.1`?

Reply:
ah, I wanted to use `dev` will update ASAP. Thank you 😄